code/modules/mob/living/carbon/human/species/species.dm

```diff
@@ -84,6 +84,9 @@
 	var/virus_immune
 	var/can_revive_by_healing				// Determines whether or not this species can be revived by simply healing them
 
+	//Death vars.
+	var/death_message = "seizes up and falls limp, their eyes dead and lifeless..."
```

Comment:
code\modules\mob\living\carbon\human\species\species.dm:88:error: "seizes up and falls limp, \his eyes dead and lifeless...": macro \his requires preceeding expression
Apparently not.